docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  app:
+    image: python:3.10-slim
+    volumes:
+      - ./src:/app
+    working_dir: /app
+    env_file:
+      - ./src/.env
```

Comment:
Could you supply a sample .env file? Just so we know what params are needed...

Im thinking either sample.env or document it in the readme

Reply:
yeah i can do that